geopandas/io/arrow.py

```diff
@@ -57,15 +57,17 @@ def _create_metadata(df):
     for col in df.columns[df.dtypes == "geometry"]:
         series = df[col]
         column_metadata[col] = {
-            "crs": series.crs.to_wkt() if series.crs else None,
             "encoding": "WKB",
+            "geometry_type": list(Series(series.geom_type.unique()).dropna()),
             "bbox": series.total_bounds.tolist(),
```

Comment:
It might be useful to make it optional to compute the "geometry_type" (if not calculated, we can set it to "Unknown"), but since right now the optional "bbox" is also always included (which is more costly to calculate), that's something we can tackle later / together.